### PR TITLE
Move Oleville to the news section

### PR DESCRIPTION
**Pull request by elijahverdoorn**
Oleville view was really news, and I like having an even number of front-page buttons. Closes part of #652.

### CODE REVIEW

File: views/news/tabs.js [hunk 1]
```diff
@@ -14,15 +15,11 @@ export default [
...
-    rnVectorIcon: {iconName: 'megaphone'},
-    component: NewsContainer,
-    props: {
-      url: 'http://oleville.com/politicole/feed/',
-      mode: 'rss',
-      name: 'PoliticOle',
-    },
+    id: 'OlevilleView',
+    title: 'Oleville',
+    rnVectorIcon: {iconName: 'mouse-pointer'},
```
Comment (hawkrives) [inline views/news/tabs.js]: `mouse-pointer` is not a valid icon for the tab bar, apologies. Tab bar uses ionicons, while the homescreen uses Entypo.

Comment (elijahverdoorn) [reply views/news/tabs.js]: That's what I get for developing on android. Fixed.

File: views/news/tabs.js [hunk 2]
```diff
@@ -14,15 +15,11 @@ export default [
...
-    rnVectorIcon: {iconName: 'megaphone'},
-    component: NewsContainer,
-    props: {
-      url: 'http://oleville.com/politicole/feed/',
-      mode: 'rss',
-      name: 'PoliticOle',
-    },
+    id: 'OlevilleView',
+    title: 'Oleville',
+    rnVectorIcon: {iconName: 'ion-mouse'},
```
Comment (hawkrives) [inline views/news/tabs.js]: Sadly, it has to be an icon that's available in an `ios-` variant.

See https://github.com/StoDevX/AAO-React-Native/blob/master/views/components/tabbed-view/index.ios.js#L35 – the tab bar loads `ios-${name}-outline` and `ios-${name}` variants.

You also won't need the `ion-` prefix, as rn-vector-icons removes that.

Comment (drewvolz) [reply views/news/tabs.js]: Changing the icon to `happy` which matches the icon used in `calendar`

**Review by drewvolz — APPROVED** [review]

**Review by hawkrives — APPROVED** [review]

**Comment by hawkrives** [comment]
If you want a different icon, @elijahverdoorn, glance through http://ionicframework.com/docs/v2/ionicons/ and use the name in the left column.

The `happy` icon was chosen because it's what we were using in the Calendar screen already.

**Pull request merged**